spec/date.html

```diff
@@ -13,20 +13,22 @@ <h1>The Temporal.Date Constructor</h1>
       <li>is the intrinsic object <dfn>%Temporal.Date%</dfn>.</li>
       <li>creates and initializes a new Temporal.Date object when called as a constructor.</li>
       <li>is designed to be subclassable. It may be used as the value of an `extends` clause of a class definition. Subclass constructors that intend to inherit the specified Temporal.Date behaviour must include a super call to the %Temporal.Date% constructor to create and initialize subclass instances with the necessary internal slots.</li>
+      <li>has a `"length"` property whose value is 4.</li>
```

Comment:
Let's drop this - it should derive from the definition.

Reply:
I did it because of precedent on the other constructors in 262, but I will remove if you feel strongly against it.

Reply:
I do feel strongly :)

Reply:
Done.